server/db/index.js

```diff
@@ -1,23 +1,21 @@
 //this is the access point for all things database related!
 
-const db = require('./db')
+const db = require("./db");
 
-const User = require('./models/User')
-const Address = require("./models/Address")
-const Product = require('./models/Product');
-
-
-Address.belongsTo(User)
-User.hasOne(Address)
+const User = require("./models/User");
+const Address = require("./models/Address");
+const Product = require("./models/Product");
```

Comment:
Don't forget to import your models onto server/db/index.js and add them to module.exports object. Also add associations here as well.